### DISCUSSION TITLE
Serenity.Extension  installed package  but still couldn't found  in reference

### DISCUSSION THREAD

**Discussion by Nsj94** [serenity-is/Serenity#5984] [Help]
Help needed

**Comment by VictorTomaili** [comment]
do you run this command? 

```cmd
dotnet sergen restore
```

**Reply by Nsj94** [reply]
No i. didn't run this command . Actually i am working on this template first time Now in the Movie table Tutorial There is a step regarding install serenity.extension and remove some ts file i did that and created my own  ts file. . But even after installing this package it still shows  it doesnt exist ..
